packages/design-system/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -80,6 +80,10 @@ export class ChoiceList extends React.PureComponent {
   }
 }
 
+ChoiceList.defaultProps = {
```

Comment:
Originally I had hoped to have the `errorPlacement` default occur inside `FieldContainer`, but there are several areas where the higher level components need to access this prop to properly handle the bottom placed error. I.e. Dropdown, TextField, and Autocomplete (eventually)

Reply:
This means the flag will have to be used in several areas instead of one, but I'm not sure if theres another solution here.

Reply:
So does this mean that the setting of the defaults will need to happen in the components themselves instead of the higher level `FieldContainer` ?

Reply:
yep pretty much. Once we make the flag it will be the same either way for whoever is setting the default but it means there are multiple components using the flag rather than just 1